unittests/builtinhttp/builtinhttp.cpp

```diff
@@ -426,6 +426,38 @@ private Q_SLOTS:
         }
     }
 
+    // Test that that if the setUseWSAddressing is set int the client  then the SOAP action
```

Comment:
`that that`? ;)

`int` -> `in`

two spaces before "then"

Reply:
Fixed